packages/odie-client/src/components/closed-conversation-footer/index.tsx

```diff
@@ -20,13 +30,17 @@ export const ClosedConversationFooter = () => {
 		} );
 	};
 
-	if ( chat.status !== 'closed' ) {
+	if ( chat?.status !== 'closed' && currentSupportInteraction?.status !== 'solved' ) {
 		return null;
 	}
 
 	return (
 		<div className="odie-closed-conversation-footer">
-			<span>{ __( 'This conversation has been completed', __i18n_text_domain__ ) }</span>
+			{ chat?.status === 'closed' && (
+				<span className="odie-closed-conversation-footer__content">
+					{ __( 'This conversation has been completed', __i18n_text_domain__ ) }
+				</span>
+			) }
```

Comment:
Do we still need this, now that we show the "Chat with support ended" tag and there is no more input?

Reply:
Let's do it!